chromium_src/content/browser/private_aggregation/private_aggregation_manager.cc

```diff
@@ -0,0 +1,21 @@
+/* Copyright (c) 2024 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "content/browser/private_aggregation/private_aggregation_manager.h"
+
+#define GetManager GetManager_Unused
```

Comment:
there's no reason to include the original file here, it's just making the override more complicated

Reply:
Fixed in 68ce1fb3a03f3cb1fa9eb909384d547ac04b0d23